src/styles/getReportActionContextMenuStyles.js

```diff
@@ -0,0 +1,63 @@
+import CONST from '../CONST';
+import styles from './styles';
+import variables from './variables';
+import themeColors from './themes/default';
+
+/**
+ * Generate styles for the buttons in the mini comment actions menu.
+ *
+ * @param {String} [buttonState] - One of {'default', 'hovered', 'pressed'}
+ * @returns {Array}
+ */
+function getMiniButtonStyle(buttonState = CONST.BUTTON_STATES.DEFAULT) {
+    const defaultStyles = [styles.p1, styles.mv1, styles.mh2, {borderRadius: variables.componentBorderRadius}];
```

Comment:
Can we actually make the `borderRadius` something like `variables.componentBorderRadius - 1` so that the border radius of the inner items lines up better with the outer radius? It would look like this:
![Uploading image.png…]()

Reply:
mmmmm, love that idea Shawn!